CHANGELOG.md

```diff
@@ -24,6 +24,12 @@ All notable changes to this project will be documented in this file.
 ## Unreleased
 * ### ALL
     * #### Added
+        * Support for nested repeated capabilities. This allows things like
```

Comment:
Still says neested. There may be other places?

Reply:
Fixed, along with test names. That was all I found still using nest[ed]